deeprank/learn/NeuralNet.py

```diff
@@ -191,8 +191,6 @@ def __init__(self, data_set, model,
         elif self.task == 'class':
```

Comment:
chain1 chain2 must be change
Line 38- save_hitrate= False?

Reply:
I changed it: https://github.com/DeepRank/deeprank/commit/0768e89e68c411b40fa6f031fa8fa9998119d0ec